src/raster/r.survey/r.survey.html

```diff
@@ -0,0 +1,227 @@
+containing absolute elevation values with respect to a vertical datum 
+(the same datum used by the input DEM). Observer height (respect to the 
+ground, the default is 1.75 m) and maximum distance of observation 
+(default is 1000 m)  are parameters needed by the r.viewshed module. 
+The object radius is used to approximate, with an equivalent circle, 
+the minimum size of an object centered in the cells center and oriented 
+according to the slope and aspect of the cells. Since r.survey produces 
+different types of outputs, a name to be used for the prefix of the 
+output maps is requested.  
+
+<p> 
+<p>
+
+3D Distance is the three 
+dimensional linear distance between a viewpoint and a target pixel (see 
+the illutratio bellow). The min3dDistance map portrays  the value of 
+the minimum three-dimensional distance between each pixel and the 
+closest viewpoint, in meters.
+<p>  
+
+<img src="r.survey_figures/fig_2.png" alt="e3Ddistance">  
+
+<p> 
+Given a 
+viewpoint, r.survey calculate the unit vector describing direction and 
+sense of the line of sight between that viewpoint and each visible 
+pixel (see illustration bellow). We define View Angle as the angle 
+between the unit vector representing the line of sight and the unit 
+vector normal to the terrain surface in each pixel. The maxViewAngle 
+map shows the value of the maximum View Angle between each pixel and 
+the viewpoints, in degrees. It is a measure of the most frontal view 
+each single cell is visible from.  View angle output is always larger 
+than 90&deg; and smaller than 180&deg;. 
+<p>  
+
+<img src="r.survey_figures/View_Angle.png" alt="view angle">   
+
+<p> 
+
+Solid Angle is one of the best and most objective indicators for quantifying 
+the visibility. The idea is that any observed object will be 
+progressively less appreciable the more far away and the more tilted it 
+is with regard to the viewpoint. As a consequence Solid Angle depends 
+on the size of the observed object as well as on the distance and the 
+orientation from where this object is observed. Solid Angle of a 
+surface is, by definition, equal to the projected spherical surface in 
+the evolving sphere divided by the square of the radius of the sphere. 
+As an aproximation, we use the ellipse surface area in place of the 
+projected spherical surface in the evolving sphere. The maxSolidAngle 
+map shows the value of the maximum Solid Angle (expressed in 
+steradians) among those measured from the different viewpoints. In this 
+map, the closest pixels to the observation points have to be 
+interpreted with special attention, considering that the error, respect 
+to the real Solid Angle, can reach until the 10% in the immediate 
+neighbor pixels. For reference: 1 sr is equal to approximately 3283 
+(deg<sup>2</sup>) square degrees and to 1.18 · 10<sup>7</sup> square 
+minutes (min<sup>2</sup>). It means that 2 · 10<sup>-6</sup> sr ≃ 23.6 
```

Comment:
```suggestion
(deg<sup>2</sup>) square degrees and to 1.18 · 10<sup>7</sup> square 
minutes (min<sup>2</sup>). It means that 2 · 10<sup>-6</sup> sr ≃ 23.6 
```
The `·` is invalid, please replace (two times).

Reply:
Seems this is still open, @IvanMarchesini (at least I am not sure if `·` is a valid character for the GRASS GIS parser and mkhtml tool.

Reply:
And still...